apps/website/src/components/content/SubNav.tsx

```diff
@@ -0,0 +1,36 @@
+import IconChevronRight from "@/icons/IconChevronRight";
+import { classes } from "@/utils/classes";
+import Link from "./Link";
+
+type SubNavProps = {
+  links: {
+    name: string;
+    href: string;
+  }[];
+  className?: string;
+};
+
+const SubNav = ({ links, className }: SubNavProps) => (
+  <nav
+    className={classes(
+      "sticky left-0 top-0 mt-0 bg-alveus-green-100/50 text-xl font-bold shadow-md backdrop-blur-2xl",
+      className,
+    )}
+  >
+    <div className="container mx-auto flex flex-row items-center gap-2 p-1 px-2">
```

Comment:
```suggestion
    <div className="container mx-auto flex flex-row items-center gap-2 py-1 px-2">
```
Avoid having classes that are overwriting each other